validator.py

```diff
@@ -84,6 +84,12 @@ def __init__(self, filename, train_mode=False):
             for x in self.dep_json['trainingOnlySupplies']:
                 self.allowed_supplies.remove(x)
 
+            for character in self.loaded_yaml.get('state', {'characters': []})['characters']:
+                if not character.get('has_blanket', False):
```

Comment:
I agree with Darren. I had thought that I was getting the error correctly, but double-checked and I think that a "not" got put in this line by accident. I _think_ it should be if character.get('has_blanket', False), because we want to error if they _do_ have a blanket, and default to False if they don't so that it doesn't end up in this if statement

Reply:
I saw this issue as well and fixed this.